qiita_db/metadata_template.py

```diff
@@ -1561,9 +1561,16 @@ def update(self, md_template):
                                % ', '.join(samples_diff))
         columns_diff = set(new_map.columns) - set(current_map.columns)
         if columns_diff:
-            raise QiitaDBError('The new sample template differs from what is '
-                               'stored in database by these columns names: %s'
-                               % ', '.join(columns_diff))
+            queue = 'add_cols_%d' % self._id
```

Comment:
A general comment: using the approach of multiple queues on a single function, we still can face the problem in which half of the function is successfully applied while the other half is not, leaving the system in a inconsistent state. Can you use a single queue in the entire function?

Thanks!